core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityManager.scala

```diff
@@ -42,6 +43,10 @@ class EntityManager(providerBuilders: Set[EntityProviderBuilder[_ <: EntityProvi
 
   def resolveProvider(requestArguments: EntityRequestArguments): Try[EntityProvider] = {
 
+    if (!WorkspaceType.RawlsWorkspace.equals(requestArguments.workspace.workspaceType)) {
+      throw new DataEntityException(s"This functionality only available to ${CloudPlatform.GCP} workspaces.")
+    }
+
```

Comment:
This is the heart of the PR: throw an error for non-`RawlsWorkspace` workspaces.